src/ResourceManager/Insights/Commands.Insights/Commands.Insights.csproj

```diff
@@ -41,23 +41,29 @@
     <DelaySign>true</DelaySign>
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
+  <PropertyGroup>
```

Comment:
Why is this section added?  If you want seoparate settings for the debug build,  they should be placed inside a conditional, so they don't apply to debug and release.